ember/app/templates/components/settings-panels/profile.hbs

```diff
@@ -89,6 +89,13 @@
       </div>
     {{/if}}
 
+      <div class="form-group">
+          <label class="control-label">{{t "allowed-to-upload-flights"}}</label>
+          <br>
+            {{input type="checkbox" checked=uploadPermissionClub }}
+          {{t "club-members"}}
+      </div>
```

Comment:
see http://getbootstrap.com/css/#checkboxes-and-radios for the right way to label checkboxes with Bootstrap 😉